meta/hofff/contao-consent/en.yml

```diff
@@ -0,0 +1,18 @@
+en:
+    title: Contao Consent
+    description: >
+        With our extension for the declarations of consent you can easily and legally secure obtain the required agreements of your users. We provide you lawyer-tested sample texts in German and English for Google Webfonts, Google Analytics, Matomo, YouTube, Vimeo and Google Maps (further services and languages will follow). Furthermore, you can also integrate your own tools and scripts to obtain consent. With our freely available Contao Consent Bridge, other extensions can also provide their own consent settings and pass them to Contao Consent.
```

Comment:
```suggestion
        With our extension for the declarations of consent you can easily obtain the required agreements of your users in a legally compliant way. We provide awyer-tested sample texts in German and English for Google Webfonts, Google Analytics, Matomo, YouTube, Vimeo and Google Maps (further services and languages will follow). Furthermore, you can also integrate your own tools and scripts to obtain consent. With our freely available Contao Consent Bridge, other extensions can also provide their own consent settings and pass them to Contao Consent.
```

Reply:
Just to high light what's different, since GitHub is not showing that:

```diff
-you can easily and legally secure obtain the required agreements of your users
+you can easily obtain the required agreements of your users in a legally compliant way
```
```diff
-We provide you lawyer-tested sample texts
+ We provide lawyer-tested sample texts
```